rootdir/etc/init.target.rc

```diff
@@ -27,6 +27,8 @@
 #
 #
 
+import /vendor/etc/init/init.mi_thermald.rc
```

Comment:
Can remove this
Should start on its own, no need to import.

Reply:
I'm not so sure to be honest so I kept it there. Should be fine I guess.

Reply:
https://github.com/ArrowOS-Devices/android_device_xiaomi_vayu/commit/6311b55ea1498aeafd0aa26275c2e24752eab99e

Reply:
It starts on its own, so putting an import here will cause it to split out as a duplicate definition

Reply:
Done! Checked my logs too and this `init: /vendor/etc/init/init.mi_thermald.rc: 8: ignored duplicate definition of service 'mi_thermald'` appears! Did you checked if abnormal high temperature is fixed for you?

Reply:
I don't usually face abnormal high temperature in my usage

Reply:
> I don't usually face abnormal high temperature in my usage

I am facing a bit high temps on normal usage too hope this commit fixes it

Reply:
> > I don't usually face abnormal high temperature in my usage
> 
> I am facing a bit high temps on normal usage too hope this commit fixes it

If you are building yourself you can include the pathset and report back.

Reply:
> > > I don't usually face abnormal high temperature in my usage
> > 
> > 
> > I am facing a bit high temps on normal usage too hope this commit fixes it
> 
> If you are building yourself you can include the pathset and report back.

Is the minimal cpu set commit done git reset i was unable to find it on arrow device org tree